cip-TOKEN-STD/cip-meiersi-Canton-Network-Token-Standard.md

```diff
@@ -0,0 +1,567 @@
+be funneled through the apps' settlement worklows.
+
+CC is standards compliant except for the following two limitations:
+
+- *short submission delay*: the CC implementation only supports a 1 minute
+  delay between preparing a transaction and submitting it to the network.
+  This is in contrast to the 24h delay targeted by the standard.
+- *no holding fees in portfolio view*: the amounts of CC holdings always show
+  the amount as of the round that they were created in. Wallets cannot
+  use the standardized APIs to display holding fees for CC holdings, but they
+  can read the `Amulet` contracts themselves if they desire to show holding fees.
+
+Both of these limitations are expected to be removed in the future, as
+explained in [Canton Coin Limitations](#canton-coin-limitations).
+
+
+### Details
+
+The standard defines six APIs. Each consists of an on-ledger Daml API and
+optionally an off-ledger HTTP API. The six APIs and their purpose are:
+
+- *token metadata API*: serve symbol name, total supply, URL for registry-specific UI, other metadata
+- *holdings API*: populate portfolio view
+- *transfer instruction API*: initiate and monitor FOP transfers
+- *allocation API*: allocate assets to execute DVP transfers
+- *allocation request API*: uniform way for apps to request allocations
+- *allocation instruction API*: uniform way for wallets to create allocations
+
+A draft PR with all Daml and OpenAPI definitions for all six APIs is linked
+below in the [Reference implementation](#reference-implementation) section. Daml
+APIs are specified using Daml interfaces and HTTP APIs are specified using
+OpenAPI.
+
+The following diagram provides an overview of the interactions enabled by these
+APIs.
+
+![API interaction diagram](images/api-diagram.png)
+
+
+#### Implementation Requirements
+
+The code in the [Reference implementation](#reference-implementation) provides
+the exact definitions of the APIs together with inline comments specifying the
+exact contracts between API clients and implementors.
+
+Asset registries can in principle freely choose which of the five APIs
+concerning them they want to implement. However, they should implement all five
+of them to maximize the utility of their tokens. The exception are asset
+registries whose authoritative holding records are not maintained on Canton.
+They may decide to not implement the "holdings API" to avoid stale holding
+records on Canton.
+
+Wallets and custody solutions should aim to make use of all six APIs to provide
+a uniform UX across all Canton Network tokens. They should query a registry's
+metadata to figure out which APIs it supports and adjust their UI accordingly.
+
+Apps may use the "allocation API" to orchestrate asset transfers as part of
+their own workflows. They may also implement the "allocation request API"
+to integrate with wallets in a uniform way.
+
+##### Global Synchronizer Connectivity for Settlement Workflows
```

Comment:
Wondering if this section makes sense in the CIP. At this point it's mostly irrelevant as there is only one synchronizer so I don't think it matters for getting it accepted.

And it doesn't make any statement that requires standardization. So I'd lean towards just omitting it and covering this kind of stuff through app dev docs.

Reply:
fair, let's keep it then.

Reply:
somehow I can't click resolve conversaiton probably becuase it's your repo so feel free to resolve yourself